.github/CODEOWNERS

```diff
@@ -40,4 +41,6 @@ main_fr.tex @Gastonia02 @qwrtln @Heegu-sama
 **/ru.po @ohgodnotagain @qwrtln @Heegu-sama
 /translations/assets/ru/ @ohgodnotagain @qwrtln @Heegu-sama
 main_ru.tex @ohgodnotagain @qwrtln @Heegu-sama
-#...
+#
+# Ukrainian
+**/ua.po @SerhiiSUr
```

Comment:
Please make it similar to all the other languages. As for the owners, there should be yourself, myself, csagataj2 and Heegu-sama.

Reply:
Done